pkg/cli/utils/apimodel.go

```diff
@@ -283,9 +260,9 @@ func MisconfigurationSeverityToAPIMisconfigurationSeverity(sev misconfigurationT
 	}
 }
 
-func ConvertMisconfigurationResultToAPIModel(misconfigurationResults *misconfiguration.Results) (*models.MisconfigurationScan, error) {
+func ConvertMisconfigurationResultToMisconfigurations(misconfigurationResults *misconfiguration.Results) (*[]models.Misconfiguration, *[]string, error) {
```

Comment:
`ConvertMisconfigurationResultToMisconfigurationsAndScanners` maybe? To keep it consistent with what we have above like `ConvertMalwareResultToMalwareAndMetadata` and below `ConvertInfoFinderResultToInfosAndScanners `